test/functional/test_framework/mintlayer/utxo.py

```diff
@@ -0,0 +1,202 @@
+import substrateinterface
```

Comment:
This file is also old. Is the plan to just merge this to staging and updated it later to be up to date with `utxo-tester`?